Count-word-occurrences/main_test.go

```diff
@@ -0,0 +1,26 @@
+package main
+
+import (
+	"reflect"
+	"testing"
+)
+
+func TestWordOccurrences(t *testing.T) {
```

Comment:
Other useful tests:
 * all unique words
 * multiple occurrences of the same word
 * a non-space separator